ruby-monday-blog/db/migrate/20150818035637_add_name_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddNameToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :first_name, :string
+    add_column :users, :last_name, :string
```

Comment:
I think you can do this in postgres `ALTER TABLE <table_name> ADD CONSTRAINT <contraint_name> CHECK (first_name is NOT NULL and last_name is NOT NULL and role = 1);`